tracer/test/Datadog.Trace.IntegrationTests/LibDatadog/TraceExporterTests.cs

```diff
@@ -0,0 +1,162 @@
+// <copyright file="TraceExporterTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Datadog.Trace.Agent;
+using Datadog.Trace.Agent.DiscoveryService;
+using Datadog.Trace.AppSec.Rasp;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.IntegrationTests.LibDatadog;
+
+public class TraceExporterTests
+{
+    [SkippableTheory]
+    [InlineData(TestTransports.Tcp)]
+    [InlineData(TestTransports.Uds)]
+    [InlineData(TestTransports.WindowsNamedPipe)]
+    public async Task SendsTracesUsingDataPipeline(TestTransports transport)
+    {
+        if (EnvironmentTools.IsWindows() && Environment.Is64BitProcess == false)
+        {
+            throw new SkipException("Can't use data pipeline on win-x86, there is no good way to copy native libraries for win-x86");
```

Comment:
😕 I think this comes down to the mistake in the copy files build. It definitely should work